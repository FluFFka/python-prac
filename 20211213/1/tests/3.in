[-100000000000000000000000000000000000000000000000000000, 1000000000000000000000000000000000000000000000, 0.5, 0.1, 0.11, True, *list(range(100, 110))]
