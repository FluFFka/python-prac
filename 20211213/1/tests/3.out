[-100000000000000000000000000000000000000000000000000000, 0.1, 0.11, 0.5, True, 100, 101, 102, 103, 104, 105, 106, 107, 108, 109, 1000000000000000000000000000000000000000000000]
